opensuse-tomcat-image/src/main/docker/Dockerfile

```diff
@@ -75,6 +91,15 @@ WORKDIR $CATALINA_HOME
 HEALTHCHECK CMD curl --fail http://localhost:8081/healthcheck || exit 1
 
 EXPOSE 8080 8081 8443
-CMD ["catalina.sh", "run"]
+
+###
+# Start container
+# Entrypoint script to run tomcat as non-root user
+###
+COPY entrypoint.sh /
+RUN chmod +x /entrypoint.sh
+
+ENTRYPOINT ["/entrypoint.sh"]
```

Comment:
This is overwriting the existing entrypoint so it will cause the startup scripts to be bypassed.